src/app/components/transcriptions/approve-transcript/approve-transcript-buttons/approve-transcript-buttons.component.html

```diff
@@ -51,5 +58,5 @@ <h1 class="govuk-fieldset__heading">Do you approve this request?</h1>
 
 <div class="govuk-button-group">
   <button class="govuk-button" data-module="govuk-button" (click)="onSubmit()">Submit request</button>
```

Comment:
This should say 'Submit' instead of 'Submit request'

Reply:
Fixed